index.html

```diff
@@ -3,11 +3,14 @@
   <head>
     <meta charset="utf-8">
     <title>did:onion Method Specification</title>
-    <!-- <script src="https://www.w3.org/Tools/respec/respec-w3c" async class="remove"></script> -->
+    <script src="https://www.w3.org/Tools/respec/respec-w3c" async class="remove"></script>
     <script class="remove">
       var respecConfig = {
-        specStatus: "PRE-DRAFT",
+        previousPublishDate: "2020-11-20",
+        previousMaturity: "UNOFFICIAL",
+        specStatus: "UNOFFICIAL",
```

Comment:
Note the use of "UNOFFICIAL" to address concerns raised on  https://github.com/w3c-ccg/community/issues/171